examples/job/get-file-from-job.ipynb

```
@@ -4,65 +4,37 @@
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {
-    "id": "_eLv7xE68GIo"
+    "id": "q1kKqnws8GIq"
    },
-   "source": [
-    "<a href=\"https://colab.research.google.com/github/Exabyte-io/api-examples/blob/dev/examples/job/get-file-from-job.ipynb\" target=\"_parent\">\n",
-    "<img alt=\"Open in Google Colab\" src=\"https://user-images.githubusercontent.com/20477508/128780728-491fea90-9b23-495f-a091-11681150db37.jpeg\" width=\"150\" border=\"0\">\n",
-    "</a>"
-   ]
+   "source": "# Get-File-From-Job\n\nThis example demonstrates how to use Mat3ra RESTful API to check for and acquire files from jobs which have been run. This example assumes that the user is already familiar with the [creation and submission of jobs](create_and_submit_jobs.ipynb) using our API.\n\n> <span style=\"color: orange\">**IMPORTANT NOTE**</span>: In order to run this example in full, an active Mat3ra.com account is required. Alternatively, Readers may substitute the workflow ID below with another one (an equivalent one for VASP, for example) and adjust extraction of the results (\"Viewing job files\" section). RESTful API credentials shall be updated in [settings](../../utils/settings.json).\n\n\n## Steps\n\nAfter working through this notebook, you will be able to:\n\n1. Import [the structure of Si](https://materialsproject.org/materials/mp-149/) from Materials Bank\n2. Set up and run a single-point calculation using Quantum Espresso.\n3. List files currently in the job's directory\n4. Check metadata for every file (modification date, size, etc)\n5. Access file contents directly and print them to console\n6. Download files to your local machine\n\n## Pre-requisites\n\nThe explanation below assumes that the reader is familiar with the concepts used in Mat3ra platform and RESTful API. We outline these below and direct the reader to the original sources of information:\n\n- [Generating RESTful API authentication parameters](../system/get_authentication_params.ipynb)\n- [Creating and submitting jobs](../job/create_and_submit_job.ipynb)"
```

Ruff (0.15.15)

[error] `await` statement outside of a function

(F704)